src/osmf/js/osmf_flash.js

```diff
@@ -90,7 +90,7 @@
     }
     this.encodings = testForFlash();
     this.technology = "flash";
-    this.features = []; //when CC need to be enabled, it needs to be set to OO.VIDEO.FEATURE.CLOSED_CAPTIONS
+    this.features = ["closedCaptions"];
```

Comment:
Do you have html5-common loaded in this file?  I would recommend using OO.VIDEO.FEATURES.CLOSED_CAPTIONS